src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private final int colorValues = Color.values().length;
```

Comment:
1. better name
2. it will be better to place it as method-level constant
```suggestion
    private final int COLORS_AMOUNT = Color.values().length;
```